src/Compilers/VisualBasic/Portable/Analysis/FlowAnalysis/AbstractFlowPass.vb

```diff
@@ -2528,6 +2528,19 @@ lUnsplitAndFinish:
             Visit(node.Expression)
             Return Nothing
         End Function
+
+        Public Overrides Function VisitInterpolatedStringExpression(node As BoundInterpolatedStringExpression) As BoundNode
+            For Each item In node.Contents
+                Visit(item)
+            Next
+
+            Return Nothing
+        End Function
+
+        Public Overrides Function VisitInterpolation(node As BoundInterpolation) As BoundNode
+            Visit(node.Expression)
+            Return Nothing
```

Comment:
I added logic to visit them. It doesn't hurt anything and, as I said, if we decide to relax the restriction in the future the logic will already be correct. The C# implementation already follows this pattern.